src/impact-metrics/context.ts

```diff
@@ -0,0 +1,17 @@
+import { CustomHeaders } from '../headers';
```

Comment:
I've given this its own home because I think we need to do a bit more here than we're currently doing. The custom headers function is designed to allow users to hot swap their API key at run time. We're not currently respecting that and we should but I don't want to do that in this PR